### DISCUSSION TITLE
How to enable "if the task fails, restart every..." feature?

### DISCUSSION THREAD

**Discussion by petercoin** [dahall/TaskScheduler#918] [Q&A] [answered]
I have set both of RestartCount and RestartInterval for the task but after installing it, the option ""if the task fails, restart every..."" is still not be enabled.
How can I enable it programmatically?

**Comment by dahall** [comment, marked as answer]
Set `TaskDefinition.Settings.RestartInterval` to a value greater than or equal to 1 minute and then set `RestartCount` to a value greater than 0.
